composer.json

```diff
@@ -20,37 +20,9 @@
         "symfony/phpunit-bridge": "^4.0"
     },
     "scripts": {
-        "fast-install" : [
-            "php scripts/check.php",
-            "php bin/console doctrine:database:create --if-not-exists",
-            "php bin/bin/doctrine dbal:import claroline.sql -vvv",
-            "php bin/console assets:install public --symlink",
-            "php scripts/data-symlink.php"
-        ],
-        "sync": [
-            "composer fetch",
-            "composer build",
-            "php bin/console claroline:update --env=prod",
-            "composer disable-maintenance"
-        ],
-        "sync-dev": [
-            "composer fetch-dev",
-            "composer build",
-            "php bin/console claroline:update -vvv",
-            "composer disable-maintenance"
-        ],
-        "fetch": [
-            "composer install --prefer-dist --no-dev --optimize-autoloader",
-            "npm install"
-        ],
-        "fetch-dev": [
-            "composer install --prefer-source",
-            "npm install"
-        ],
         "build": [
             "Claroline\\CoreBundle\\Library\\Installation\\Refresher::deleteCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "npm run webpack"
+            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap"
```

Comment:
This seems to print:
`Class Sensio\Bundle\DistributionBundle\Composer\ScriptHandler is not autoloadable, can not call build script`
Is this expected?

Reply:
There is an error because the underlying package has been removed. 
I will just remove it.